code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -1,25 +1,40 @@
-/obj/item/weapon/gun/projectile/automatic //Hopefully someone will find a way to make these fire in bursts or something. --Superxpdude //Except burstfire isn't fit for an rp server --Mark
+/obj/item/weapon/gun/projectile/automatic //This should never be spawned in, it is just here because of code necessities. 
+	name = "daka SMG"
+	desc = "A small SMG. You really shouldn't be able to get this gun. Uses 9mm rounds."
+	icon_state = "c05r"	//Used because it's not used anywhere else
+	load_method = SPEEDLOADER
+	ammo_type = /obj/item/ammo_casing/a9mm
+	projectile_type = /obj/item/projectile/bullet/pistol
+
+//Burst is the number of bullets fired; Fire delay is the time you have to wait to shoot the gun again, Move delay is the same but for moving after shooting. .

```

Comment:
These comments are unnecessary, as the code performed here is done on the base gun, rather than on automatics.

Reply:
Someone said I should put them in anyway